book/src/SUMMARY.md

```diff
@@ -12,6 +12,7 @@
   - [Syntax aware motions](./syntax-aware-motions.md)
   - [Keymap](./keymap.md)
   - [Commands](./commands.md)
+  - [Static Commands](./static-commands.md)
```

Comment:
On second thought let's keep both commands in `commands.md` and use a table of contents like we have for `book/src/keymap.md` and have one section for typable commands and the next for static commands.

So `book/src/commands.md` can become:

```markdown
## Commands

- [Typable commands](#typable-commands)
- [Static commands](#static-commands)

## Typable commands

Typable commands are used from command mode and may take arguments. Command mode can be activated by pressing `:`. The built-in commands are:

{{#include ./generated/typable-cmd.md}}

## Static commands

Static commands take no arguments and can be bound to keys. Static commands can also be executed from the command picker (`<space>?`). The built-in static commands are:

{{#include ./generated/static-cmd.md}}
```